integrations/buildsystem/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-library`
+    id("org.jetbrains.kotlin.jvm")
+    id("org.jetbrains.dokka")
+}
+
+Deployment.initialize(project)
```

Comment:
If you're adding a new module - it should be published as well. I don't think we need to create a separate jar for this though